src/components/view.js

```diff
@@ -41,6 +41,13 @@ export class View {
    */
   init(unusedDialog) {}
 
+
```

Comment:
I'd hope no changes to the `View` interface would be needed.

Reply:
`view` interface has signature, `init(unusedDialog)` and for `smartButton` we do not have `dialog` param to pass.

Reply:
`View` is still defined as a "view" inside a dialog. So using it w/o a dialog does not seem to be like a right use.

Reply:
OK. Reverting this change.
Making direct call to `activityPort.openIframe` now.